{
  "name": "large-image-source-mapnik",
  "version": "1.16.2",
  "requires_dist": [
    "large-image",
    "large-image-source-gdal",
    "mapnik",
    "importlib-metadata",
    "girder-large-image"
  ],
  "error": "{\"type\": 5, \"message\": \"Failed to install: Command '['/tmp/pypi/large-image-source-mapnik/bin/python3', '-m', 'pip', 'install', 'large-image-source-mapnik']' returned non-zero exit status 1.\"}"
}